app/views/spree/shared/_afterpay_messaging.html.erb

```diff
@@ -10,4 +10,6 @@
   ></script>
 <% end %>
 
-<%= content_tag("afterpay-placement", nil, data: data) %>
+<% unless ::SolidusAfterpay::PaymentMethod.active.first.excluded_product?(product) %>
```

Comment:
could you move the `unless` condition to include also the `content_for :head` stuff? I don't think we need to include the js if we don't use the messaging

Reply:
Makes sense! Done!